src/Main.java

```diff
@@ -1,5 +1,60 @@
+import java.util.Arrays;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+//Задачи 1, 2
+        int[] weight = {1, 2, 3};
```

Comment:
здесь нужно было создать массив вот так:
```suggestion
        int[] weight = new int[3];
```
и затем заполнить его элементы (как написано в подсказе в задании)